assets/js/components/SearchBar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import debounce from 'lodash.debounce';
+import { createUseStyles } from 'react-jss';
+import { useForm } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { DEBOUNCE_SEARCH_MS } from '../utils/constants.js';
+import { Icon } from 'design-react-kit';
+
+const useStyle = createUseStyles({
+  icon: {
+    minWidth: '40px',
+    minHeight: '40px',
+  },
+  searchBar: {
+    color: 'var(--primary) !important',
+    fontSize: '3rem',
+    height: 'auto !important',
+    width: '100%',
+    padding: '.375rem .75rem !important',
+    marginBottom: 0,
+  },
+});
+
+export const SearchBar = React.memo(({ defaultValue = '', placeholder = '', onChange }) => {
+  const { register } = useForm({
+    defaultValues: {
+      search: defaultValue,
+    },
+  });
+  const classes = useStyle();
+
+  const handleOnChangeSearchValue = debounce((e) => {
+    onChange(e.target.value);
+  }, DEBOUNCE_SEARCH_MS);
+
+  return (
+    <h2 className="d-flex align-items-center mb-0">
+      <Icon color="primary" icon="it-search" className="icon-lg mb-0" />
+      <input
```

Comment:
I would use design-react-kit component as much as we can, in this way we will get all the benefits in term of accessibility and style from the kit itself

Reply:
Yes, you are right, but sometimes I feel a bit frustrated to use the proxy pattern. What I mean is that to me, it seems overkill to create a <Button> component if I simply want a <button> element, managed natively by the browser.

Moreover, probably because I'm not used to using it yet, sometimes it adds classes that in developers.italia don't match the desired ui, and so I have to add a more specialized style that void the effect of the standard classes.

Reply:
got your point, I would like, however, to avoid having to cascade all the changes made upstream in the kit for common and urgent issues such as accessibility, what do you think?

Reply:
Yes, you're right, the accessibility is a problem. We'll treat it in a different PR.